### PR TITLE
[bug fix] NPU triton ops

### PR DESCRIPTION
**Pull request by MichelleWu351**

### CODE REVIEW

File: mojo_opset/tests/accuracy/operators/test_sampling.py [hunk 1]
```diff
@@ -39,9 +39,28 @@ def test_topp_sampling(shape, topk, topp, min_tokens_to_keep):
...
+        ref_probs_list.append(ref_p.float())
+        npu_probs_list.append(npu_p.float())
+    ref_all = torch.cat(ref_probs_list, dim=1)
+    npu_all = torch.cat(npu_probs_list, dim=1)
+    ref_mean = ref_all.mean(dim=1, keepdim=True)
+    npu_mean = npu_all.mean(dim=1, keepdim=True)
+    ref_std = ref_all.std(dim=1, keepdim=True)
+    npu_std = npu_all.std(dim=1, keepdim=True)
+    torch.testing.assert_close(npu_mean, ref_mean, atol=1e-2, rtol=1e-2)
+    torch.testing.assert_close(npu_std, ref_std, atol=1e-2, rtol=1e-2)
```
[the remark this answers is not in the document]
Comment (MichelleWu351) [reply mojo_opset/tests/accuracy/operators/test_sampling.py:64]: applied suggestion.

File: mojo_opset/tests/accuracy/operators/test_sampling.py [hunk 2]
```diff
@@ -52,15 +71,19 @@ def test_topp_filter(shape, topk, topp, min_tokens_to_keep):
...
+    torch.testing.assert_close(
+        npu_probs.float(), ref_probs.float(), atol=1e-2, rtol=1e-2
     )
 
+    ref_idx_sorted, _ = ref_idx.sort(dim=-1)
+    npu_idx_sorted, _ = npu_idx.sort(dim=-1)
+    assert torch.equal(ref_idx_sorted, npu_idx_sorted), (
+        f"Selected token indices do not match! "
+        f"Expected: {ref_idx_sorted}, Got: {npu_idx_sorted}"
+    )
```
[the remark this answers is not in the document]
Comment (MichelleWu351) [reply mojo_opset/tests/accuracy/operators/test_sampling.py:89]: applied suggestion.

**Review by kevin-hongkai — APPROVED** [review]
Fix master branch on 950 npu
